realm/realm-library/src/androidTest/java/io/realm/DynamicRealmTests.java

```diff
@@ -338,6 +338,7 @@ public void findFirstAsync() {
                 .between(AllTypes.FIELD_LONG, 4, 9)
                 .findFirstAsync();
         assertFalse(allTypes.isLoaded());
+        looperThread.keepStrongReference.add(allTypes);
```

Comment:
If this indeed fixes its we should have a ton of bugs in all other unit tests using the `@RunTestInLooperThread`?
`allTypes` do go out of scope when exiting the method though 😢